### ISSUE TITLE
WINAPI physical key SendInput does not register repeating key when held 

### ISSUE THREAD

**Issue by pattontim** [kozec/sc-controller#685]
According to the WINAPI reference, setting KEYEVENTF_SCANCODE and defining wScan should simulate a physical keystroke. However with this arrangement (ran thru mingw32) only one input is detected by programs. The programs detect a single key input on key down only, but the correct key held behaviour is detected in the MINGW32 window and other linux windows. How can this be remedied on windows?
```
        INPUT input;
	if (scc_keycode_to_win32_scan(key) == 0) {
		// Invalid keycode
		return;
	}
	DDEBUG("WINAPI PRESS %u, %d", scc_keycode_to_win32_scan(key), pressed);
	
	input.type = INPUT_KEYBOARD;
	input.ki.time = 0;
	input.ki.wVk = 0;
    input.ki.dwExtraInfo = 0;
	input.ki.dwFlags = (pressed ? (KEYEVENTF_SCANCODE) : (KEYEVENTF_SCANCODE | KEYEVENTF_KEYUP));
	uint16_t scancode = scc_keycode_to_win32_scan(key);
	if (scancode & 0xE000) {
		input.ki.dwFlags |= KEYEVENTF_EXTENDEDKEY;
		input.ki.wScan = scancode & 0x00FF;
	} else {
		input.ki.wScan = scancode;
	}
 
	UINT r = SendInput(1, &input, sizeof(input));
```

**Comment by kozec** [comment]
Well, most obvious "fix" would be to emulate keypresses on timer.

But I don't have same issue on my (virtual) Windows. Isn't issue caused by input handling of that specific application?

**Comment by pattontim** [comment]
> Well, most obvious "fix" would be to emulate keypresses on timer.
> 
> But I don't have same issue on my (virtual) Windows. Isn't issue caused by input handling of that specific application?

Are you running an emulator running windows 10? I am seeing this issue on Windows 11 across the entire OS

**Comment by kozec** [comment]
Yeah, Windows 10. I really hope not even Microsoft is crazy enough to make this kind of change between versions.
